hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,17 @@
+//import { useEffect, useState } from "react";
+
+// const useLocalStorage = (key: string, defaultState: string | null) => {
+//   const [value, setValue] = useState(localStorage.getItem(key) || defaultState);
+
+//   useEffect(() => {
+//     localStorage.setItem(key, JSON.stringify(value));
+//   }, [value]);
+
+//   return [value, setValue];
+// };
+
+const useLocalStorage = (key: string, defaultState: string) => {
+  localStorage.setItem(key, JSON.stringify(defaultState));
+};
+
```

Comment:
요건 훅이라고 보기 어려운 것 같아요! 따라서 use라는 네이밍은 오히려 헷갈릴 수 있을 것 같네요~

Reply:
로컬스토리지의 입장에서는 값을 받는 것뿐이지 그 값이 어떤 값인지를 알 필요는 없을 것 같아요~
```suggestion
const useLocalStorage = (key: string, value: string) => {
  localStorage.setItem(key, JSON.stringify(value));
};

```

Reply:
그리고 로컬스토리지를 다루는 모듈을 만들때 key값을 은닉하는 방식으로 추상화를 해보면 좋을 것 같습니다!